base/steer/FairRun.h

```diff
@@ -222,6 +228,10 @@ class FairRun : public TNamed
 
     Bool_t                   fMarkFill; //!
 
+    FairAlignmentHandler fAlignmentHandler;
+
+    void AlignGeometry() const;
+
     ClassDef(FairRun ,4)
```

Comment:
Shouldn't we increase the schema version, too?

Reply:
For consistancy YES, technically we never save the FairRun so it does not heart but it it shows the reader that something change.

Reply:
Ah yes we will also update that